packages/data/src/components/use-select/index.js

```diff
@@ -104,7 +105,100 @@ const renderQueue = createQueue();
  * ```
  * @return {UseSelectReturn<T>} A custom react hook.
  */
+function Selecter( registry ) {
+	const NOTHING = {};
+	let lastMapSelect = NOTHING;
+	let lastMapResult = NOTHING;
+	let lastMapResultValid = false;
+
+	const NULLSCRIBER = () => {
+		lastMapResultValid = false;
+		return () => {};
```

Comment:
Should this return `NOTHING` as well?

```suggestion
		return () => NOTHING;
```

Reply:
No, this function is an "unsubscribe" function that does nothing.

Reply:
I see, we might want to use a comment to explain the subtle difference between "nothing" and `NOTHING` 😅